From 69c4c922bebfa5fae67ecb2c6a2553a6f08b14f9 Mon Sep 17 00:00:00 2001
From: William Wu <william.wu@rock-chips.com>
Date: Mon, 22 Jun 2020 17:17:02 +0800
Subject: [PATCH] ARM: configs: disable UAC1 UAC2 UVC and RNDIS for
 rv1126-evb-ddr3-v10-tb.config

This patch disable USB Gadget functions UAC1/UAC2/UVC/RNDIS,
and only support F_FS for ADB.

Change-Id: If04bb18f9c620299dde4b2452e418e63badeec12
Signed-off-by: William Wu <william.wu@rock-chips.com>
---
 arch/arm/configs/rv1126-evb-ddr3-v10-tb.config | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/arch/arm/configs/rv1126-evb-ddr3-v10-tb.config b/arch/arm/configs/rv1126-evb-ddr3-v10-tb.config
index 9623b28ae8d6..15b724ef4171 100644
--- a/arch/arm/configs/rv1126-evb-ddr3-v10-tb.config
+++ b/arch/arm/configs/rv1126-evb-ddr3-v10-tb.config
@@ -53,13 +53,15 @@ CONFIG_SOUND=m
 CONFIG_STMMAC_ETH=m
 CONFIG_STMMAC_PLATFORM=m
 # CONFIG_TEE is not set
+# CONFIG_USB_CONFIGFS_F_UAC1 is not set
+# CONFIG_USB_CONFIGFS_F_UAC2 is not set
+# CONFIG_USB_CONFIGFS_F_UVC is not set
+# CONFIG_USB_CONFIGFS_RNDIS is not set
 CONFIG_USB_DWC3=m
 CONFIG_USB_DWC3_OF_SIMPLE=m
 CONFIG_USB_DWC3_ROCKCHIP_INNO=m
 CONFIG_USB_EHCI_HCD=m
 CONFIG_USB_EHCI_HCD_PLATFORM=m
-CONFIG_USB_F_UAC1=m
-CONFIG_USB_F_UAC2=m
 CONFIG_USB_HID=m
 # CONFIG_USB_KBD is not set
 CONFIG_USB=m
@@ -68,7 +70,6 @@ CONFIG_USB=m
 CONFIG_USB_OHCI_HCD=m
 CONFIG_USB_OHCI_HCD_PLATFORM=m
 CONFIG_USB_STORAGE=m
-CONFIG_USB_U_AUDIO=m
 CONFIG_USB_XHCI_HCD=m
 CONFIG_USB_XHCI_PLATFORM=m
 CONFIG_WL_ROCKCHIP=m
-- 
2.35.3

